pdb2pqr/main.py

```diff
@@ -809,6 +810,8 @@ def main():
     parser = build_main_parser()
     args = parser.parse_args()
     _ = main_driver(args)
+    if _ == 1:
```

Comment:
Shouldn't this be:
```
  if 1 == main_driver(args)
      sys.exit(1)
```
Do we need the `_` unused variable?

Reply:
We're not currently doing anything with `_`, so that ought to be ok.  I'll change it in a moment